Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/AnomalyDescriptor.java

```diff
@@ -40,9 +40,14 @@
     // information
     DiVector attribution;
 
+    long numberOfUpdates;
```

Comment:
We use a field name `totalUpdates` in `UpdateCoordinator`. Can we use the same field name here?

Reply:
Done